src/main/java/mate/academy/model/CinemaHall.java

```diff
@@ -0,0 +1,56 @@
+package mate.academy.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+
+@Entity
+public class CinemaHall {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private int capacity;
+    private String description;
+
+    public CinemaHall() {
```

Comment:
Don't create a default constructor when it is not needed. In this case, the default constructor is not being used and Hibernate can use the parameterized constructor or reflection to instantiate the object. Consider removing the default constructor if it's not required by the framework or for other purposes.